spyder/plugins/completion/languageserver/plugin.py

```diff
@@ -574,15 +575,20 @@ def generate_python_config(self):
         }
 
         # Jedi configuration
+        extra_paths = self.get_option('spyder_pythonpath', section='main',
+                                      default=[])
         if self.get_option('default', section='main_interpreter'):
             environment = None
         else:
             environment = self.get_option('custom_interpreter',
                                           section='main_interpreter')
+            # jedi can miss the site-package: see
+            # https://github.com/davidhalter/jedi/issues/1540
+            extra_paths += get_python_site_packages(environment)
```

Comment:
If this is happening  only on non conda envs, we should probably do this only on those cases?

Reply:
Agreed